src/middlewared/middlewared/plugins/pool.py

```diff
@@ -709,6 +723,7 @@ def pool_extend(self, pool):
         'pool_create', 'pool_update',
         ('rm', {'name': 'name'}),
         ('rm', {'name': 'encryption'}),
+        ('rm', {'name': 'deduplication'}),
```

Comment:
Why this?

Reply:
It is a dataset attribute and `pool.update` does not do anything with it